python/python/glide/config.py

```diff
@@ -479,6 +506,8 @@ class GlideClusterClientConfiguration(BaseClientConfiguration):
                 ]
 
         use_tls (bool): True if communication with the cluster should use Transport Level Security.
+                If you need to configure an insecure TLS setup (for example, bypassing certificate validation),
```

Comment:
You can change `use_tls` from bool to union of bool and enum. So TLS configuration will have the same (and the only) entry point without breaking changes

Reply:
Moved to use TLS advanced config class as in the near future will add support to send key and cert to the client etc.